### PR TITLE
DOCS-981: Refactor update viam-server

### PR DESCRIPTION
**Pull request by andf-viam**
Moves existing dedicated update `viam-server` content to live on existing manage `viam-server` page.
- Removed instructions for updating on non-stable branch, since we no longer present local install guidance, or discuss branch selection elsewhere in the docs.

### CODE REVIEW

File: docs/installation/manage/_index.md [hunk 1]
```diff
@@ -147,6 +147,47 @@ brew services restart viam-server
 {{% /tab %}}
 {{< /tabs >}}
 
+## Update `viam-server`
```
Comment (npentrel) [inline docs/installation/manage/_index.md:150]: Make sure to add an alias to the to the old update page then I think this is good to go

**Review by npentrel — APPROVED** [review]
Add alias then good to go

File: docs/installation/manage/_index.md (hunk 1 above)
Comment (andf-viam) [reply docs/installation/manage/_index.md:150]: Oops, thank you!! On it.

**Review by Otterverse — CHANGES_REQUESTED** [review]
Two small requests.

**Review by Otterverse — APPROVED** [review]
LGTM!

**Pull request merged**